setup.cfg

```diff
@@ -24,8 +24,8 @@ classifiers =
 packages = find:
 python_requires = >=3.6
 install_requires =
-    google-cloud-logging>=1.6.0,<2.0.0
-    google-cloud-resource-manager>=0.28.3
+    google-cloud-logging
```

Comment:
Probably should make this `>=2.0.0` now? The changes look like they would break the old version.